### ISSUE TITLE
Miss new Single Axis Solar Tracker

### ISSUE THREAD

**Issue by Computerhenne** [tolwi/hassio-ecoflow-cloud#127]
Hi, I bought two EcoFlow Solar Tracker and the installation, app integration into Ecoflow cloud went well. Now I would like integrate this in my smart home and so far the HACS integration works but I cannot intgrate the trackers using 'Diagnostic'. Its planned or possible to integrate such soilar trackers too?

**Comment by Nid01** [comment]
Where did you take the needed device SN from?
It's not available for me in the app. so from the devices directly?

**Comment by Computerhenne** [comment]
Yes, directly from the device control unit. There is a number S/N: HZ31 ... 16 digits.

**Comment by foxthefox** [comment]
First we have to find the suitable data for integration.
I have made a small writup how to connect to the mqtt broker via node-red:
https://gist.github.com/foxthefox/11ac35433c228085a8f5e25d5651632a
If you can make such a setup, it would be great.

After going to the APP to look at data of the tracker in debug4 something should be logged, that should be postet here (needed for displaying values)
If there are commands from the APP possible, triggering them would cause messages in debug5, that should be also postet here (needed for commands from HA)
